imqsauth/config.go

```diff
@@ -75,13 +73,12 @@ func (x *Config) IsContainer() bool {
 
 func (x *Config) GetHostname() string {
 	if x.hostname == "" {
-		if x.HostnameFile != "" {
-			hostname_b, err := ioutil.ReadFile(x.HostnameFile)
-			if err != nil {
-				hostname_b, _ = ioutil.ReadFile(filepath.Join(filepath.Dir(x.lastFileLoaded), x.HostnameFile))
-			}
-			x.hostname = strings.TrimSpace(string(hostname_b))
+		hostname_b, exists := os.LookupEnv("IMQS_HOSTNAME_URL")
+		if !exists {
+			hostname_b = ""
```

Comment:
I think you should set `x.hostname = ""` here rather than hostname_b. Then you can put the part that does 
`x.hostname = strings.TrimSpace(string(hostname_b))` in an else statement. Something like 

```
	hostname_b, ok := os.LookupEnv("IMQS_HOSTNAME_URL")
	if !ok {
		x.hostname = ""
	} else {
		x.hostname = strings.TrimSpace(string(hostname_b))
	}
```

Reply:
Agreed.